### ISSUE TITLE
[Linux] Telegram requests focus after a notification

### ISSUE THREAD

**Issue by FrankAlbella** [telegramdesktop/tdesktop#7964]
### Steps to reproduce
1. Have Telegram unfocused 
2. Receive a message notification 
3. "Telegram is ready" notification is sent by Gnome

### Expected behaviour
Telegram should not request focus after sending a notification 

### Actual behaviour
Telegrams requests that the app be focused. This results in two notifications per received message, and can interfere with workflows using the [Steal My Focus](https://extensions.gnome.org/extension/234/steal-my-focus/) gnome extension. 

### Configuration
**Operating system:** Pop!_OS 20.04
**Version of Telegram Desktop:** 2.1.7
**Installation source (Linux Only)** - the official website / GitHub releases / flatpak / snap / distribution package: Flatpak (from Flathub)
**Used theme**:  Tinted (Night Mode)

**Comment by Aokromes** [comment]
not a bug, it can be disabled with one setting, iirc somewhere on notifications.

**Comment by teo1978** [comment]
Yet the default of that setting should be not to request focus.

Seriously, this was already mentioned in another bug and you need to change that. One shouldn't have to look for settings and tweak them just to get the behavior that should be normal.

Requesting focus for a received message is a stupid default. That's what notifications are for. Some users with very peculiar use cases (e.g. their workflow is mainly focused on chatting via telegram with priority over anything else on their computer) may want this, so it's great that there is a setting. But the default should be disabled.

Correct defaults for settings are important, they shouldn't be an afterthought. "There's a setting you can change so who cares", no. The default should correspond to the most sensible behavior for the most common usage scenario and/or widely adopted conventions, and in this case there's no possible debate about what the most sensible default is: disabled. Unless you can provide a reasoning against that.

**Comment by bviktor** [comment]
Agreed, it's rather odd to receive TWO notifications for messages by default. But now at least I know how I can disable it.

For the lazy:

- Settings / Notifications / Draw attention to the window <- uncheck
- Settings / Notifications / Use native notifications <- check
